drafter/history.py

```diff
@@ -188,17 +188,17 @@ class VisitedPage:
     button_pressed: str
     original_page_content: Optional[str] = None
     old_state: Any = None
-    started: datetime = dataclass_field(default_factory=datetime.utcnow)
+    started: datetime = dataclass_field(default_factory=lambda:datetime.now(timezone_UTC))
     stopped: Optional[datetime] = None
 
     def update(self, new_status, original_page_content=None):
         self.status = new_status
         if original_page_content is not None:
-            self.original_page_content = format_page_content(original_page_content, 120)
+            self.original_page_content = format_page_content(original_page_content, 120).replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
I think that it would be better to use [`html.escape`](https://docs.python.org/3/library/html.html#html.escape) here instead?